.gitignore

```diff
@@ -75,7 +75,11 @@ Thumbs.db
 .yarn/install-state.gz
 
 # nektos/act - Github Actions
-bin/
+/bin/
```

Comment:
We shouldn't ignore all `bin` dirs in the repo.  It's especially annoying if you have tools that obeys .gitignore.